code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -202,17 +191,20 @@
 	desc = "A flask of holy water...it's been sitting in the Necropolis a while though."
 	list_reagents = list("hell_water" = 100)
 
-/obj/item/reagent_containers/food/drinks/holyoil
+/obj/item/reagent_containers/food/drinks/bottle/holyoil
```

Comment:
I made them NOT bottles for a reason : /
I would like them to be non-useable as a weapon as well as when thrown, not have a icon.

Reply:
Listen up, you blamed me on discord upon someone's else report of said flask disappearing upon being thrown.
Even thought that's the sort of average behaviour for non-bottle drink containers with the IsGlass var TRUE, it'd still be fairly confusing for some.
And after all, I don't see much of an issue with them being a bottle subtype.

Reply:
*No* I blamed you for the wording on being usable for slipping rather then just being a good throwing weapon. Too different issues, I even said to MrWhite it was MENT to have no flask once thrown and that I would prefer it to dquel itself once broken.

Reply:
Tribly, I wrote `"sipped"` there, not `"slipped"`.
It's a different verb which roughly means "drinking little at a time", basically the antonym of `"chugging"`, nothing to do with clowns.

Reply:
I... Cant read. My bad